### DISCUSSION TITLE
Is pubDatetime filled in manually every time?

### DISCUSSION THREAD

**Discussion by whcxxb** [satnaing/astro-paper#239] [FAQ] [answered]
Looking forward to the answer

**Comment by satnaing** [comment]
Every time?
If you are creating a blog post for the first time, yes. It is required if you are not integrating AstroPaper with some kind of HeadlessCMSes (eg: [DecapCMS](https://decapcms.org/)).
And you won't have to add the `pubDatetime` again for that post in the future.

**Comment by SSmale** [comment, marked as answer]
Is there a way to have this added by git and/or CI? same with the updated date?

https://mademistakes.com/notes/adding-last-modified-timestamps-with-git/ Looks like there is a hook to do it, should be able to get the (A) for an added and (M) for update. 

I will try to get it working and add a PR for this in. custom snippets for the template would be good too so that the front matter is always there.

**Reply by satnaing** [reply]
Hello @SSmale 
I'll mark this as the answer. 
Feel free to update your comment for further clarification and precision.

**Comment by SSmale** [comment]
there are 2 ways now to get the new post date, either by the front matter snippet or the git hook, see the PRs above. Hope that helps

**Comment by satnaing** [comment]
Related PR has been merged.
However, I think this issue should be in the discussion and thus I'll convert this to discussion.
